lang/java/java.talon

```diff
@@ -35,11 +33,8 @@ action(user.code_operator_less_than_or_equal_to): " <= "
 action(user.code_operator_and): " && "
 action(user.code_operator_or): " || "
 action(user.code_operator_bitwise_and): " & "
-action(user.code_operator_bitwise_and_equals): " &= " 
```

Comment:
Is there a reason you removed the code_comment and the bitwise operators? I believe Java supports all of these?

Reply:
It is supported Indeed. But I only want to add a small set of Java commands that are Widely used. I don't consider this widely used. This can always be added later.